CORE/elements/text_field.rb

```diff
@@ -1,6 +1,4 @@
-
-
-class TextFieldElement < CoreElement
+class TextFieldElement < FillableElement
```

Comment:
Remove the fillableElement classing please. this is not saving us any work right now, it is just adding another class to keep track of.